lib/internal/modules/esm/hooks.js

```diff
@@ -114,6 +116,23 @@ class Hooks {
     this.addCustomLoaders(userLoaders);
   }
 
+  /**
+   * Import and register custom/user-defined module loader hook(s).
+   * @param {string} urlOrSpecifier
+   */
+  async register(urlOrSpecifier) {
+    const moduleLoader = require('internal/process/esm_loader').esmLoader;
+
+    const parentURL = pathToFileURL(getCwdSafe()).href;
+    const keyedExportsSublist = await moduleLoader.import(
+      [urlOrSpecifier],
```

Comment:
```suggestion
      urlOrSpecifier,
```

Reply:
@jlenon7 Antoine's PR has landed. Do you have time to rebase this branch onto the latest of `main` and apply this correction?

Reply:
I've already done it 🙌